lib/discovery_api/schemas/visualizations.ex

```diff
@@ -21,4 +21,11 @@ defmodule DiscoveryApi.Schemas.Visualizations do
       visualization -> {:ok, visualization}
     end
   end
+
+  def update(visualization_changeset), do: update(visualization_changeset, [])
+
+  defp update(visualization_changeset, opts) do
```

Comment:
Can you just make opts default to [] and then delete the above line?
```suggestion
  defp update(visualization_changeset, opts \\ []) do
```

Reply:
yeah, that's better